src/backend/distributed/executor/multi_task_tracker_executor.c

```diff
@@ -96,7 +97,8 @@ static TaskExecStatus ManageTaskExecution(TaskTracker *taskTracker,
 										  Task *task, TaskExecution *taskExecution);
 static TransmitExecStatus ManageTransmitExecution(TaskTracker *transmitTracker,
 												  Task *task,
-												  TaskExecution *taskExecution);
+												  TaskExecution *taskExecution,
+												  uint64 *totalIntermediateResultSize);
```

Comment:
Consider this optional: It'd be kind of nice if we passed down some more generic data structure (e.g. `DistributedExecutionStats *stats`) in which we keep the `totalIntermediateResultSize`, such that we can easily add other counters in the future without having to change signatures everywhere.